src/main/groovy/edu/oregonstate/mist/api/InfoResource.groovy

```diff
@@ -14,6 +14,21 @@ import javax.ws.rs.core.Response
 class InfoResource extends Resource {
     private Info info = new Info()
 
+    public InfoResource() {
+        def stream = this.class.getResourceAsStream('/build.properties')
+        if (stream == null) {
+            throw new Exception("couldn't load build.properties")
+        }
```

Comment:
* (!stream) has different semantics

* yes, the program will not start if build.properties (or resource.properties) is not found. but that isn't anything new; the only difference is it used to throw a FileNotFoundException. do we really want to try to start up without the build info? now that it's bundled in the jar it shouldn't really ever be missing